.github/workflows/build-release.yml

```diff
@@ -3,7 +3,7 @@ name: '[CI] Build Release'
 on:
     push:
         tags:
-        -   '[0-9][0-9][0-9][0-9].[0-9][0-9].[0-9]'
+        -   '[0-9][0-9][0-9][0-9].[0-9][0-9].[0-9]+'
```

Comment:
Just leaving the comment for posterity.. As agreed in previous discussions, this complies with the chosen format `YYYY.MM.MICRO`, where MICRO is a counter.